styles/typography.scss

```diff
@@ -0,0 +1,61 @@
+@import "./global-css-settings";
+@import "./content-spacing";
+
+// font family
+$header-primary-font: var(--title-font-family);
+$text-primary-font: var(--font-family);
```

Comment:
This is a mixture of documentation components and design tokens.
I think we should be really clear that documentation components should not be using vibe tokens.
They should take all relevant design from the `vibe-storybook-components`